src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Startup.cs

```diff
@@ -28,9 +28,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseBlazorDebugging();
             }
 
-            app.UseMvc(routes =>
+            app.UseRouting();
+
+            app.UseAuthentication();
+            app.UseAuthorization();
```

Comment:
Why have `UseAuthentication`/`UseAuthorization` in this template by default? Developers would certainly have to take further steps before these would do anything useful, so they might as well add these themselves at the point where they are setting up auth.

Reply:
This might be a bad copy-paste on my part, I'm happy to take them out.